.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
we might also need to add npm-debug.log* 
We will add the other values when we add more features